### DISCUSSION TITLE
When is the stable comprehensive release?

### DISCUSSION THREAD

**Discussion by oogxdd** [electric-sql/electric#1702] [Questions]
I'm building a local-first multiplatform app (with React Native and, likely, Tauri)

I'm considering choosing your library, but the docs are just too empty now. I know you had a more comprehensive release, but I doubt I should start using this with the legacy version.

When we can expect to have a comprehensive stable release again, with React-native and Tauri/Electron examples? 

Also, here are couple of things I'm concerned:

- How will migrations take effect? If I make a migration on my remote databases - will local databases apply that migration as well?

- Will mutations I made on my local db be applied to the remote db when coming online again? Or does it only work with querying? Or I should utilize another library like Tanstack query?

- How conflicts would be handled (eg I modified the same object on different devices when they were offline)?

Honestly right now I'm leaning more towards PowerSync / Evolu / RXdb as they seem to be more established at this point of time.

**Comment by KyleAMathews** [comment]
Hey we're in alpha so if you need a more mature solution then you should try another tool. Some of your questions might be answered by reading this post on the goals of the project https://electric-sql.com/blog/2024/07/17/electric-next

**Reply by beorn** [reply]
Is the [roadmap](https://github.com/electric-sql/archived-electric-next/milestones) referenced in the blog authoritative? It seems a bit sparse and is in the `archived-electric-next` repo.

I was also looking into Electric for a local-first use case, and it would be nice to have a github issue to subscribe to for updates on when reasonable local-write support is implemented.
